frontend/src/features/admin-form/common/components/CollaboratorModal/CollaboratorWizardContext.tsx

```diff
@@ -40,12 +42,17 @@ const useCollaboratorWizardContext = (): CollaboratorWizardContextReturn => {
     [],
   )
 
+  const handleForwardToRemoveSelf = useCallback(() => {
```

Comment:
Just checking on the name: what does the `ForwardTo` mean in this function name? I had the same question for `handleForwardToTransferOwnership()`, but forgot to ask earlier.